From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 30 Jan 2018 16:52:52 +0100
Subject: Drop rpath from examples

---
 meson.build | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/meson.build b/meson.build
index 55b07ef..ac9fb73 100644
--- a/meson.build
+++ b/meson.build
@@ -25,10 +25,6 @@ add_project_arguments(
 	'-I@0@'.format(meson.build_root()),
 	language: 'c',
 )
-add_project_link_arguments(
-	'-Wl,-rpath,@0@'.format(meson.build_root()),
-	language: 'c',
-)
 
 conf_data = configuration_data()
 
